RedHatInsights/insights-advisor-frontend
# Add support for cypress-based tests

Implements https://issues.redhat.com/browse/RHIF-71.

This adds support for the cypress component and e2e tests (version 10). Also utilizes functions and constants from `@redhat-cloud-services/frontend-components-utils`.

## How to test

Run `npm run test` (will run together with jest tests) or `npm run test:ct` (cypress tests alone). You should obtain the results of RulesTable.cy.js test.
